### ISSUE TITLE
The details of the hyperparameters of training procedure

### ISSUE THREAD

**Issue by NatureGeorge** [AngxiaoYue/ReQFlow#3]
Hi! Brilliant works. I am wondering how long you have been training the QFlow (as well as ReQFlow). In other words, what is your total number of training epochs? Although I see the `max_epochs` is set to be `1000`, it seems too large. Is there any metric used as the convergence indicator, and did you use something like early stopping? I see no clues in the configurations, and it is crucial to make sure that the models are well-trained, so I would appreciate it if you could provide more details about it. 

All the best,
Zefeng Zhu

**Comment by AngxiaoYue** [comment]
Hi Zefeng, Thanks for your interest in our work! The `max_epochs=1000` follows the default setting of [FrameFlow](https://github.com/microsoft/protein-frame-flow). In particular, after observing loss convergence (around 600 epochs on PDB), we select checkpoints based on the metrics of the generated protein validation set. We choose the checkpoint where the ca_ca_valid_percent > 0.99 and the proportions of secondary structures are closest to the dataset's average values. For example, when selecting the checkpoint of QFlow on PDB, the generated protein validation set has a `ca_ca_valid_percent = 0.996`, a `helix_percent = 0.400`, and a `strand_percent = 0.283`. As for ReQFlow, very small epochs are needed for the reflow operation.

**Comment by NatureGeorge** [comment]
Thanks for your reply. That makes sense. :-)
